pages/api-reference/funding-sources/create-funding-source-for-customer.mdx

```diff
@@ -13,13 +13,26 @@ There are two methods available for adding a bank or credit union account to a C
 
 Before a Dwolla account or Dwolla API Customer is eligible to transfer money from their bank or credit union account they need to verify ownership of the account, either via Instant Account Verification (IAV) or micro-deposits. For more information on bank account verification, reference this [funding source verification](https://developers.dwolla.com/concepts/bank-funding-source#bank-addition--verification-methods) resource article.
 
+[Verified Customers](https://developers.dwolla.com/concepts/customer-types#customer-types) also have the ability to add Virtual Account Numbers, or `VANs`. A `VAN` can only be used outside of the Dwolla Platform to route ACH payments both to and from the Dwolla Balance.
+
+<AlertBar variation="warning">
+  Because VANs behave as a route to a Dwolla Balance, only{" "}
+  <a href="/concepts/customer-types#verified-customer">Verifed Customers</a> and
```

Comment:
small misspell in Verified Customers

Reply:
Dang! Good catch! This [commit](https://github.com/Dwolla/dev-portal/pull/260/commits/3dc2ff6c0ce9227251c3fafcc48415ac2f28f9dc) should fix it!